### DISCUSSION TITLE
passeggiate di peso massimo

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/progettazione-di-algoritmi#67] [Esercizi A-L]
Dato un grafo $G$ **diretto**, scrivere lo pseudocodice di un algoritmo che trova la passeggiata di peso massimo usando al massimo $k$ archi in $O(k(n + m))$

**Comment by Traster-uni** [comment]
![WhatsApp Image 2024-06-13 at 5 04 46 PM](https://github.com/sapienzastudentsnetwork/progettazione-di-algoritmi/assets/84441861/3f107567-4a35-4c9f-9504-97aa5e6eea98)
